v2/internal/file/file.go

```diff
@@ -14,3 +17,17 @@ func Exists(filename string) (bool, error) {
 	}
 	return false, err
 }
+
+// Check if the dirPath directory exists. If not, create it
+func EnsureDirectory(dirPath string) error {
```

Comment:
I'm surprised the linter didn't flag this. The comment above an exported function needs to start with the function's name. i.e. "// EnsureDirectory does...."